web/src/js/ducks/ui/flow.js

```diff
@@ -23,10 +23,10 @@ const defaultState = {
     tab: 'request',
     content: [],
     maxContentLines: 80,
+    ctrlkey: false,
```

Comment:
Can we remove the whole ctrlkey state? I don't think we need to keep track of this in redux.